### ISSUE TITLE
question on creating complex vertical merged tab?

### ISSUE THREAD

**Issue by retsyo** [elapouya/python-docx-template#104]
in my real work situation, I have to create a complex tab which you can read in the attachment

so, is there any way to implement it in python-docx-template?

[category.docx](https://github.com/elapouya/python-docx-template/files/1631750/category.docx)

**Comment by elapouya** [comment]
I put your request to the enhancement wish list...

**Comment by FlorenceMei** [comment]
Hi, how is it going with this enhancement? i have a similar table to create and I realized that jinja style template just doesn't accept any of my tries. It seems that nested FOR loops in one table is not possible. (see my examples)
It also tried to correct the merging with docx but no luck... 
[Tests_docx_tables.docx](https://github.com/elapouya/python-docx-template/files/4869898/Tests_docx_tables.docx)

**Comment by elapouya** [comment]
it is better to transform 

```
gauche = [[G11,G12,G13],[G21,G22,G23]]
droite = [[D11,D12,D13],[D21,D22,D23]]
```

into : 

```
lignes = [[G11,G12,G13,D11,D12,D13],[G21,G22,G23,D21,D22,D23]]
```

So only one loop by row is needed.

**Comment by FlorenceMei** [comment]
Yes, I did it that way only to test if it is possible to align several loops next to each other, but apparently not. Therefore, I would like to know if there is another way to write the second table in my docx file (the table I would like to have as result)

**Comment by retsyo** [comment]
Finally, I have to launch MsWord via win32com, then merge the cells by writing python code.

I do expect there will be an easy way without opening MsWord

**Comment by tsy19900929** [comment]
https://github.com/tsy19900929/docxtpl_dynamic_vertical_merging
![test1](https://user-images.githubusercontent.com/24546045/199894258-4647b32e-820b-400c-908c-cb3abade8688.jpg)
